bats/create-tables.bats

```diff
@@ -10,41 +10,40 @@ teardown() {
 }
 
 @test "create a single primary key table" {
-    run dolt table create -s=`batshelper 1pk5col-ints.schema` test
+    run dolt sql < $BATS_TEST_DIRNAME/helper/1pk5col-ints.schema
```

Comment:
You can just delete run now and remove the $status check after because we don't inspect the $output variable.